### PR TITLE
chore: update package readmes

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: packages/components/src/components/link/docs/Angular.md [hunk 1]
```diff
@@ -22,8 +16,9 @@ import { DBLinkModule } from '@db-ui/ngx-components';
...
 ```
 
-Use component in template:
+### Use component
 
-```
-<db-link variant="primary">Link</db-link>
+```html app.component.html
+<!-- app.component.html -->
+<db-link href="#" variant="primary">Link</db-link>
```
Comment (mfranzke) [inline packages/components/src/components/link/docs/Angular.md:23]:
```suggestion
< DBLink href="#" variant="primary">Link</DBLink >
```

I thought that we wanted to use the uppercase syntax everywhere.

File: output/angular/README.md [hunk 2]
```diff
@@ -11,6 +11,63 @@ An Angular library containing all styles & components of [DB UX Design System (t
...
+```
+
+</details>
+
+## Usage
+
+```ts
+<script setup lang="ts">
+	import { DBButton } from "@db-ui/v-components";
+</script>
```
Comment (annsch) [inline output/angular/README.md:64]: This is Vue syntax, 
```suggestion
```typescript
	import { DBButton } from "@db-ui/ngx-components";
```

File: packages/components/src/components/link/docs/Angular.md [hunk 3]
```diff
@@ -1,17 +1,11 @@
 ## Angular
 
-Load SCSS globally somewhere in your app:
+For general installation and configuration look at the [ngx-components](https://www.npmjs.com/package/@db-ui/ngx-components) package.
```
Comment (annsch) [inline packages/components/src/components/link/docs/Angular.md:4]: There ist no link or explanation on how to setup CSS/SCSS for angular; shouldn't we link it?

File: packages/components/src/components/link/docs/Angular.md (hunk 1 above)
Comment (mfranzke) [reply packages/components/src/components/link/docs/Angular.md:23]: nope.

File: packages/components/src/components/link/docs/Angular.md (hunk 3 above)
Comment (nmerget) [reply packages/components/src/components/link/docs/Angular.md:4]: We need to deploy this branch to have the "installation" steps inside the npm package, which then contains the css/scss part

File: packages/components/src/components/link/docs/Angular.md (hunk 1 above)
Comment (nmerget) [reply packages/components/src/components/link/docs/Angular.md:23]: Only for vue and react, angular uses a html style

File: output/angular/README.md (hunk 2 above)
Comment (nmerget) [reply output/angular/README.md]: fixed

**Review by mfranzke — APPROVED** [review]

**Pull request merged**